### PR TITLE
add support for finding cstaticlibs and cdylibs

### PR DESCRIPTION
**Pull request by Gankra**

### CODE REVIEW

**Comment by Gankra** [comment]
Running this on top of #6 produces:

```
cargo run --release -- .\tests\projects\cargo-nonvirtual\ 

  some-cdylib@0.1.0 (no binaries)
    [cdylib] some-cdylib
  some-lib@0.1.0 (no binaries)
  some-other-lib@0.1.0 (no binaries)
  some-staticlib@0.1.0 (no binaries)
    [cstaticlib] some-staticlib
  test-bin@0.1.0 (publish = false)
    [bin] test-bin
  nonvirtual@0.1.0
    [bin] cargo-nonvirtual
    [bin] nonvirtual
```

Hooray

**Comment by Gankra** [comment]
This is necessary for https://github.com/axodotdev/cargo-dist/issues/179 which I'm not really prioritizing but figured I might as well lay the ground work for while I'm focusing on axoproject

File: src/main.rs [hunk 1]
```diff
@@ -94,6 +94,13 @@ fn print_project(project: &WorkspaceInfo) {
                 rust_binaries.push(binary);
             }
         }
+        for lib in &pkg.cdylibs {
+            eprintln!("    {}", sty.apply_to(format!("[cdylib] {}", lib)));
```
Comment (ashleygwilliams) [inline src/main.rs:98]: i would add version to this ;)

**Review by ashleygwilliams — CHANGES_REQUESTED** [review]
super small change, happy to grab the PR and add it and then merge :)

File: src/main.rs (hunk 1 above)
Comment (Gankra) [reply src/main.rs:98]: these are nested underneath the packages which specify the version, see the example output in my previous comment.

Comment (ashleygwilliams) [reply src/main.rs:98]: oh i'm dumb ignore me

**Review by ashleygwilliams — APPROVED** [review]

**Pull request merged**